### DISCUSSION TITLE
Is there any support for Prisma unsupported types?

### DISCUSSION THREAD

**Discussion by mlesin** [Brendonovich/prisma-client-rust#135] [General]
I like how Prisma organized in a way it queries data, but I really need to interact with postgres ltree type somehow

**Comment by Brendonovich** [comment]
I haven't added special handling for `Unsupported` fields, but I should probably do what is done [here](https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#unsupported-types). You're able to use raw queries to interact with unsupported types, but using the regular syntax is not possible.
